tests/test_json.py

```
@@ -1126,6 +1127,39 @@ def test_enum(self):
+        class NotMSONable:
+            def __init__(self, v1):
+                self.v1 = v1
+
+        is_m = IsMSONable(1)
+        not_m = NotMSONable(2)
+
+        is_m_jsons, is_m_map = partial_monty_encode(is_m)
+        is_m_d = json.loads(is_m_jsons)
+        assert is_m_d["@class"] == "IsMSONable"
+        assert is_m_d["v1"] == 1
+        len(is_m_map) == 0
```

Ruff (0.8.2)

Pointless comparison. Did you mean to assign a value? Otherwise, prepend `assert` or remove it.

(B015)